README.md

```diff
@@ -23,6 +23,41 @@ Service receives images, performs pixel by pixel comparison with it’s previous
 * Project - list of Builds and TestVariations
 
 ## Set up
+
+### Linux, macOS, WSL
+
+1. Install [Docker](https://docs.docker.com/get-docker/)
+2. Download the installation script
+
+```
+curl https://raw.githubusercontent.com/Visual-Regression-Tracker/Visual-Regression-Tracker/master/vrt-install.sh
```

Comment:
add `-o vrt-install.sh` otserwise it's not saved as file

Reply:
fixed